product_packaging_type_required/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Product Packaging Type Required",
-    "version": "14.0.1.1.0",
+    "version": "15.0.1.1.0",
```

Comment:
@AnnaPForgeFlow Please reset to 15.0.1.0.0

Reply:
@rousseldenis done!

Reply:
@AnnaPForgeFlow I don't think so :sweat_smile: